README.md

```diff
@@ -1,5 +1,7 @@
-loast
-=====
+<p align="center"> 
+  <img src="images/loast-logo.png" alt="Logo" width="250" height="auto">
+  <h1 align="center">LOAST</h1>
+</p>
```

Comment:
Can we convert this to markdown?
Looks like
```md
![Logo](images/loast-logo.png)

# LOAST
```
should work

Reply:
oh wait that would cancel out the centering. NVM me

Reply:
nothing else is centered though, so i'm back to thinking we should just go with markdown